### PR TITLE
New language option: Russian

### PR DESCRIPTION
**Pull request by Esinko**
Adding a new language option. Everything is now ready to be translated.

### CODE REVIEW

**Comment by Esinko** [comment]
Good job! DeveloperFromJokela can do the rest

File: index.html [hunk 1]
```diff
@@ -142,6 +158,7 @@ <h2 class="fi">.NET-kirjasto</h2>
         <div class="footer">
             <strong class="en">Notice of Non-Affiliation and Disclaimer</strong>
             <strong class="fi">Huomio vastuuvapaudestamme ja kytköksettömyydestämme</strong>
+            <strong class="ru">Уведомление о неприсоединении и отказе от ответственности</strong>
             <p class="notice en">
```
Comment (developerfromjokela) [inline index.html:163]: Laita toho Внимание eikä Уведомление

**Review by developerfromjokela — COMMENTED** [review]
I marked some things up, please correct

**Comment by developerfromjokela** [comment]
I approve, good job @ahnl

**Comment by Esinko** [comment]
Good job

**Pull request merged**